src/platforms/esp32/main/platform_nifs.c

```diff
@@ -474,7 +494,7 @@ const struct Nif *platform_nifs_get_nif(const char *nifname)
         TRACE("Resolved platform nif %s ...\n", nifname);
         return &atomvm_platform_nif;
     }
-    return NULL;
+    return ledc_nifs_get_nif(nifname);
```

Comment:
having LEDC chained in this way looks a bit hackish, it shouldn't be treated as a default/special case. On the long term I think we should use gperf here too.

Reply:
Fixed it — I think.  See what you think.